elf2uf2/elf2uf2.cpp

```diff
@@ -193,9 +282,20 @@ int pages2uf2(std::map<uint32_t, std::vector<page_fragment>>& pages, std::shared
     return 0;
 }
 
+/**
+ * @brief Converts a binary file to UF2 format.
+ * 
+ * @param in Input stream for the binary file.
+ * @param out Output stream for UF2 data.
+ * @param address Start address in the target memory.
+ * @param family_id Family ID for UF2.
+ * @param abs_block_loc Optional absolute block location.
```

Comment:
If this is optional, what special value indicates "don't use this"?

Reply:
A 0 means don't add this - any non-zero value means do use this

Reply:
Could you add this to the doxygen please @SimonWaldherr ?